src/main/java/io/milvus/client/AbstractMilvusGrpcClient.java

```diff
@@ -417,704 +164,1389 @@ private FieldData genFieldData(String fieldName, DataType dataType, List<?> obje
-                    .setAutoID(fieldType.isAutoID());
+        try {
+            // Construct CollectionSchema Params
+            CollectionSchema.Builder collectionSchemaBuilder = CollectionSchema.newBuilder();
+            collectionSchemaBuilder.setName(requestParam.getCollectionName())
+                    .setDescription(requestParam.getDescription());
+
+            for (FieldType fieldType : requestParam.getFieldTypes()) {
+                FieldSchema.Builder fieldSchemaBuilder = FieldSchema.newBuilder()
+                        .setFieldID(fieldType.getFieldID())
+                        .setName(fieldType.getName())
+                        .setIsPrimaryKey(fieldType.isPrimaryKey())
+                        .setDescription(fieldType.getDescription())
+                        .setDataType(fieldType.getDataType())
+                        .setAutoID(fieldType.isAutoID());
+
+                // assemble typeParams for CollectionSchema
+                List<KeyValuePair> typeParamsList = assembleKvPair(fieldType.getTypeParams());
+                if (CollectionUtils.isNotEmpty(typeParamsList)) {
+                    typeParamsList.forEach(fieldSchemaBuilder::addTypeParams);
+                }
 
-            // assemble typeParams for CollectionSchema
-            List<KeyValuePair> typeParamsList = assembleKvPair(fieldType.getTypeParams());
-            if (CollectionUtils.isNotEmpty(typeParamsList)) {
-                typeParamsList.forEach(fieldSchemaBuilder::addTypeParams);
+                collectionSchemaBuilder.addFields(fieldSchemaBuilder.build());
             }
 
-            collectionSchemaBuilder.addFields(fieldSchemaBuilder.build());
-        }
-
-        // Construct CreateCollectionRequest
-        CreateCollectionRequest createCollectionRequest = CreateCollectionRequest.newBuilder()
-                .setCollectionName(requestParam.getCollectionName())
-                .setShardsNum(requestParam.getShardsNum())
-                .setSchema(collectionSchemaBuilder.build().toByteString())
-                .build();
+            // Construct CreateCollectionRequest
+            CreateCollectionRequest createCollectionRequest = CreateCollectionRequest.newBuilder()
+                    .setCollectionName(requestParam.getCollectionName())
+                    .setShardsNum(requestParam.getShardsNum())
+                    .setSchema(collectionSchemaBuilder.build().toByteString())
+                    .build();
 
-        Status response;
-        try {
-            response = blockingStub().createCollection(createCollectionRequest);
+            Status response = blockingStub().createCollection(createCollectionRequest);
 
             if (response.getErrorCode() == ErrorCode.Success) {
-                logInfo("Created collection " + requestParam.getCollectionName() + " successfully!\n{}",
-                        requestParam.toString());
+                logInfo("CreateCollectionRequest successfully!");
```

Comment:
need at least the collection name here? same as the other ddl requests.

Reply:
In each api method I add a line "logInfo(requestParam.toString());", to print out the input prameters, including the collection name.

Reply:
done